packages/@webex/plugin-meetings/src/constants.ts

```diff
@@ -136,6 +136,8 @@ export const _WEBEX_MEETING_ = 'WEBEX_MEETING';
 export const _WAIT_ = 'WAIT';
 export const _MOVE_MEDIA_ = 'MOVE_MEDIA';
 
+export const _ONE_2_ONE_MEEETING_ = 'ONE_2_ONE_MEETING';
```

Comment:
I am not sure what the name should be but this still doesn't feel like something standard. We need a better name that can be documented.

Maybe, "WEBEX_DIRECT_MEETING" or so.

CC - @marcin-bazyl - Your expertise on naming is required here 😁

Reply:
Maybe the name should be "ONE_2_ONE_CALL" rather than "MEETING", because I think that's how the server teams call these things - they call them "1-1 Locus calls", they are not meetings.
What I really don't like about the already existing code is that the `type` is declared as a `string` and there is no documentation or anything to point the user of the API what values are allowed/supported for `type`. The constants currently supported are for example `_CONVERSATION_URL_` or `_LOCUS_ID_`, but these are just scattered around the codebase not even grouped together in the constants file, How is anyone supposed to know what values to pass as `type`? As you're touching this area, can we improve this and define a proper Enum for the possible values for `type` supported by the `create()` API?